WcaOnRails/app/views/competitions/edit.html.erb

```diff
@@ -122,6 +122,37 @@
         });
       </script>
 
+
+      <% if @competition_admin_view %>
```

Comment:
By "set" I meant before they put a marker on the map and choose dates from the calendar-picker.

I haven't tried yet with no date, but I'm guessing nothing will show up because of the select conditions. 

(DATEDIFF(?, CONCAT(year, '-', month, '-', day))) will probably evaluate to NULL, and NULL is not <= 90.

I guess unscheduled comps would not show up in the table, too. 

No idea about tests...